opentech/apply/dashboard/views.py

```diff
@@ -16,5 +14,12 @@ class DashboardView(SingleTableMixin, FilterView):
     filterset_class = SubmissionFilter
 
 
-class SubmissionDetailView(DetailView):
```

Comment:
Should this replace the submission detail view?

Reply:
SubmissionDetailView exists here. This shouldnt have been here.
https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/apply/funds/views.py#L9-L16

Reply:
👍 I thought it was odd